src/components/accounts/RecoverWithLink.js

```diff
@@ -117,12 +118,21 @@ class RecoverWithLink extends Component {
     }
 
     handleCopyUrl = () => {
-        const selection = window.getSelection();
-        selection.selectAllChildren(this.recoverUrl.current);
-        document.execCommand('copy');
+        if (window.matchMedia("(max-width: 767px)").matches) {
```

Comment:
@Patrick1904 I think we should better use explicit check https://developer.mozilla.org/en-US/docs/Web/API/Navigator/canShare

note that desktop Safari also has this API, but copying the link isn't available further in the flow

Reply:
We decided to only show on **mobile** because copy ux is not great on desktop as mentioned [here](https://github.com/nearprotocol/near-wallet/issues/394). Therefore, I'm adding the check for browser width for this use case. Didn't want to put it in the util function since it limits future use.

I'm currently doing `if (navigator.share)` to check for support. I can replace with `navigator.canShare(data)` if needed?

Reply:
> I can replace with navigator.canShare(data) if needed?

Actually I just checked on desktop Safari and it's unavailable, while `.share` is available. Weird.

I think might be most robust to just hardcode iPhone/Android user agent checks, as current check would have false positive if you resize window and false negative on iPad.

Reply:
Sounds good should we use [this](https://stackoverflow.com/a/21742107/5574318) method of checking mobile device OS?